study.md

```diff
@@ -66,15 +68,15 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+not, because they are not dependant on eachother
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
+yes, because the values in the_sun is equal to nearest_star
```

Comment:
"Yes" is correct, but your reasoning is incorrect.  It's because they are both references to the same object.